source/features/file-age-color.tsx

```diff
@@ -0,0 +1,47 @@
+import * as pageDetect from 'github-url-detection';
+
+import observe from '../helpers/selector-observer';
+import features from '../feature-manager';
+
+const heatMap = new Map([
+	['3', '1'],
+	['6', '2'],
+	['15', '3'],
+	['24', '4'],
+	['40', '5'],
+	['70', '6'],
+	['100', '7'],
+	['160', '8'],
+	['275', '9'],
+	['30000', '10'],
```

Comment:
We don't need these hard-coded numbers nor do we need to find a difference in "days" exactly (because JS makes that awkward to calculate).

Could you turn this into a mathematical function? This is what it should look like:

```js
function addHeatIndex(timeAgo: HTMLElement): void {
	const diff = Date.now() - new Date(timeAgo.attributes.datetime.value).getTime();
	timeAgo.dataset.rghHeat = Math.max(0, Math.min(10, Math.sqrt(diff / 1000)))
}
```

That exact "1000" number needs to be figured out. Other math functions are also welcome if they're easier to write for you. The square root looks similar to the type of curve I'm imagining.

Reply:
There are some "mathematical function finders" online where you can plug in those numbers in a x/y table and you get an approximated function you can paste here. It's probably easier than figuring out that srqt value.

Reply:
I pushed a change using your idea.

It supports up to 125 days it that better?